features/step_definitions/files_steps.rb

```diff
@@ -30,3 +30,10 @@
   expect(uncommitted_files).to eql [file_name]
   expect(IO.read file_name).to eql content
 end
+
+
+Then(/^my uncommitted file "(.+?)" is still stashed away$/) do |file_name|
+  expect(uncommitted_files).to_not include file_name
```

Comment:
Yeah, you're right. That might be simpler. It turns out git stores the untracked files differently, so we would also have to use `git show stash@{0}^3:` to see the the untracked files.